### PR TITLE
Add favicon and change font

### PR DESCRIPTION
**Pull request by Xwilarg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,3 +7,9 @@ PHP >= 8.0
 ## Features
  - Lightweight and easy to use
  - README integration
+
+## License
+
+This code is licensed under the MIT License.
+
+`deer.png` is licensed by Twitter under CC-BY 4.0.
```
Comment (TheIndra55) [inline README.md]: Not sure if this is enough or if we need to link to their repository

**Review by TheIndra55 — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (Xwilarg) [inline README.md]:
```suggestion
`deer.png` is licensed [by Twitter under CC-BY 4.0](https://github.com/twitter/twemoji#license).
```

**Review by Xwilarg — COMMENTED** [review]
Like this maybe?

File: README.md (hunk 1 above)
Comment (TheIndra55) [reply README.md]: Looks good

**Review by TheIndra55 — APPROVED** [review]

**Pull request merged**